reo/src/reopt_decomposed.jl

```diff
@@ -883,7 +883,7 @@ end
 
 function add_decomp_model(m, p::Parameter, model_type::String, mth::Int64)
 	if m[:solver_name] == "Xpress"
-		sub_model = direct_model(Xpress.Optimizer(MAXTIME=-120, MIPRELSTOP=0.03, OUTPUTLOG = 0))
+		sub_model = direct_model(Xpress.Optimizer(MAXTIME=-120, MIPRELSTOP=p.DecompOptTol, OUTPUTLOG = 0))
```

Comment:
For completeness, we should update ratioGap for Cbc and limits_gap for SCIP too.  (Likewise for MAXTIME, seconds, and limits_time if we include time limits as a user-defined option as suggested above.

Reply:
@zolanaj Can you explain if `ratioGap` and `limits_gap` are identical/equivalent to `MIPRELSTOP`, and if so we can just assign those with `p.DecompOptTol`? Relatedly with `MAXTIME` (which I agree we should have that as an input too for the subproblems), is `seconds` and `limits_time` equivalent?

Reply:
Yes,  `ratioGap` and `limits_gap` are the Cbc and Scip analogs to MIPRELSTOP and can be set using p.DecompOptTol.  Likewise for `seconds` and `limits_time` being analogous to MAXTIME for the other solvers; a single parameter should be able to populate these with the same input (e.g., `p.DecompSubTimeLimit`, or a different name if you prefer).